model/riscv_pmp_regs.sail

```diff
@@ -163,30 +164,29 @@ function pmpReadCfgReg(n) = {
 
 /* Helpers to handle locked entries */
 function pmpLockBit(cfg: Pmpcfg_ent) -> bool =
-   cfg.L() == 0b1
+  cfg.L() == 0b1
    
 function pmpLocked(cfg: Pmpcfg_ent) -> bool =
-   (cfg.L() == 0b1) & (mseccfg.RLB() == 0b0)
+  (cfg.L() == 0b1) & (mseccfg.RLB() == 0b0)
 
 function pmpTORLocked(cfg: Pmpcfg_ent) -> bool =
-   (cfg.L() == 0b1) & (mseccfg.RLB() == 0b0) & (pmpAddrMatchType_of_bits(cfg.A()) == TOR)
+  (cfg.L() == 0b1) & (mseccfg.RLB() == 0b0) & (pmpAddrMatchType_of_bits(cfg.A()) == TOR)
   
 /* Function to check whether executable region can be added or not in accordance to 4b of Smepmp spec */
 function checkExe(cfg: Pmpcfg_ent,v: bits(8)) -> Pmpcfg_ent =
-	 if ((mseccfg.MML() == 0b1) & (mseccfg.RLB() == 0b0) & (v[7] == bitone))
-   then {
-    if ((v[0] == bitzero & v[1] == bitzero & v[2] == bitone) |
-        (v[0] == bitzero & v[1] == bitone  & v[2] == bitzero)|
-        (v[0] == bitzero & v[1] == bitone  & v[2] == bitone) |
-        (v[0] == bitone  & v[1] == bitzero & v[2] == bitone))
-      then cfg                           /* If condition meets configuration remains same which means pmpcfg writes are ignored */
-    else Mk_Pmpcfg_ent(v & 0x9f)         /* Bits 5 and 6 are zero */
-    }
-   else {
-    if (mseccfg.MML() == 0b0) & (v[0] == bitzero) & (v[1] == bitone)
-      then Mk_Pmpcfg_ent(v & 0x9c)       /* Bits 5 and 6 are zero and Bits 0 and 1 are zero because of RW=01 reserved combination */
-    else Mk_Pmpcfg_ent(v & 0x9f)        /* Bits 5 and 6 are zero */
-   }
+	if ((mseccfg.MML() == 0b1) & (mseccfg.RLB() == 0b0) & (v[7] == bitone)) then {
+   if ((v[0] == bitzero & v[1] == bitzero & v[2] == bitone) |
+       (v[0] == bitzero & v[1] == bitone  & v[2] == bitzero)|
+       (v[0] == bitzero & v[1] == bitone  & v[2] == bitone) |
+       (v[0] == bitone  & v[1] == bitzero & v[2] == bitone))
+   then cfg                           /* If condition meets configuration remains same which means pmpcfg writes are ignored */
+   else Mk_Pmpcfg_ent(v & 0x9f)         /* Bits 5 and 6 are zero */
+  }
+  else {
+   if (mseccfg.MML() == 0b0) & (v[0] == bitzero) & (v[1] == bitone)
+   then Mk_Pmpcfg_ent(v & 0x9c)       /* Bits 5 and 6 are zero and Bits 0 and 1 are zero because of RW=01 reserved combination */
+   else Mk_Pmpcfg_ent(v & 0x9f)        /* Bits 5 and 6 are zero */
+  }
```

Comment:
What has actually changed here? If it is just formatting, please put it in a separate formatting commit.